packages/web-react/src/components/RadioField/RadioField.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import RadioField from './RadioField';
+import { SpiritRadioFieldProps } from '../../types';
+
+export default {
+  title: 'Components/RadioField',
+  argTypes: {
+    id: 'default',
+    isDisabled: {
+      control: 'boolean',
+    },
+    isChecked: {
+      control: 'boolean',
+    },
+    label: 'Label',
+  },
+};
+
+const Template = (args: SpiritRadioFieldProps) => <RadioField {...args} />;
+
+export const DefaultRadioField = Template.bind({});
```

Comment:
Could you please add stories as we do it in the `Alert` or `Button` component using a `stories` directory?

Also add please a group of radios to show that they are clickable. Thanks.

Reply:
done